xsd/netex_part_2/part2_journeyTimes/netex_interchange_version.xsd

```diff
@@ -159,14 +159,14 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>SCHEDULED STOP POINT at which JOURNEY MEETING takes place.</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
-			<xsd:element name="FromJourneyRef" type="VehicleJourneyRefStructure">
+			<xsd:element name="FromJourneyRef" type="JourneyRefStructure">
```

Comment:
I don't understand here : the Transmodel definition for JOURNEY MEETING is "A time constraint for one or several **SERVICE JOURNEYs** fixing interchanges between them and/or an external event (e.g. arrival or departure of a feeder line, opening time of the theatre, etc.)."

Reply:
That's right, but that would require a deprecation in 2.0

Reply:
and it should be a SERVICE JOURNEY MEETING then. And it is still not working correctly, because the whole hierarchy is messed up, because ServiceJourney is not derived from VerhicleJourney but from Journey. So by going through Journey it works.
Also: switching from quay  3 to 4 before continuing is not a ServiceJourney in any system. With Journey it still holds.

So No, I will not accept Stefan's changes.

Reply:
because this is not a feature fix, this is a bug fix name and what the hierarchy does are aligned afterwards. Your change you can do for 3.0

Reply:
@ue71603 reverted, your suggestion for the type of ConnectingJourneyRef